test/index.test.js

```diff
@@ -656,7 +680,8 @@ describe('index test', () => {
             owner: 'owner',
             repo: 'repo',
             ref: 'master',
-            scmContext: 'github:github.com'
+            scmContext: 'github:github.com',
+            refType: 'heads'
```

Comment:
Why add `refType` here? I think that the change in index.js does not affect this test case.

Reply:
Because the branch was not up to date.
so I took in origin/master

Reply:
Oh. sorry
I forgot to fix.